src/test/three-components/SmoothControls-spec.js

```diff
@@ -322,5 +324,139 @@ suite('SmoothControls', () => {
         });
       });
     });
+
+    suite('elastic over rotate', () => {
+      setup(() => {
+        controls.applyOptions({
+          overRotateAmount: 1,
+          minimumPolarAngle: QUARTER_PI,
+          maximumPolarAngle: THREE_QUARTERS_PI
+        });
+        controls.setOrbit(0, 0, 0);
+        settleControls(controls);
+      });
+
+      test('can over rotate below minimumPolarAngle', () => {
+        const minPolar = controls.options.minimumPolarAngle;
+
+        controls.setOrbit(0, minPolar, 0);
+        settleControls(controls);
+
+        expect(controls.getCameraSpherical().phi).to.be.equal(minPolar);
+
+        controls.adjustOrbit(0, QUARTER_PI * 0.25, 0);
+
+        settleControls(controls);
+
+        expect(controls.getCameraSpherical().phi).to.be.lessThan(minPolar);
```

Comment:
Shouldn't this be equal to minPolar after sufficient settling, because it springs back?